src/main/java/org/folio/innreach/domain/service/impl/InnReachRecallUserServiceImpl.java

```diff
@@ -21,6 +22,7 @@ public class InnReachRecallUserServiceImpl implements InnReachRecallUserService
   private final CentralServerRepository centralServerRepository;
   private final InnReachRecallUserRepository recallUserRepository;
   private final InnReachRecallUserMapper recallUserMapper;
+  private final CentralServerRepository centralserverRepository;
```

Comment:
repository for central server is already here

Reply:
Fixed